cmd/sebak/cmd/wallet/unfreezeRequest.go

```diff
@@ -130,7 +130,7 @@ func makeTransactionUnfreezingRequest(kpSource keypair.KP, seqid uint64) transac
 
 	txBody := transaction.Body{
 		Source:     kpSource.Address(),
-		Fee:        common.Amount(common.BaseFee),
+		Fee:        common.FrozenFee,
```

Comment:
This should be changed to `common.Amount(common.BaseFee)`

Reply:
done